community/bolt/src/docs/dev/serialization.asciidoc

```diff
@@ -420,6 +422,46 @@ Path (signature=0x50) {
 }
 ----
 
+The nodes (`N`) and relationships (`R`) are defined as follows:
```

Comment:
Specify precisely what `N` and `R` are; e.g. something like "The two lists, `N` and `R` are defined as follows" (the contents therein are specified later, so no need to say nodes and relationships in this sentence)